main.tf.json

```diff
@@ -1705,7 +1713,7 @@
                     "protect_config_version": "20.2",
                     "protect_dynamodb_version": "${var.is_dynamodb_enabled ? \"4.0\" : \"\"}",
                     "protect_ebs_version": "${var.is_ebs_enabled ? \"21.0\" : \"\"}",
-                    "protect_ec2_mssql_version": "${var.is_ec2_mssql_enabled ? \"2.2\" : \"\"}",
+                    "protect_ec2_mssql_version": "${var.is_ec2_mssql_enabled ? \"2.1\" : \"\"}",
```

Comment:
This version went down, is it correct? Is this generated from the release branch?

Reply:
Spoke with Prakash, this was an intentional change due to an unrelated EC2 MSSQL issue.